src/rewriter.ts

```diff
@@ -51,8 +51,15 @@ export abstract class Rewriter {
   visit(node: ts.Node) {
     // this.logWithIndent('node: ' + ts.SyntaxKind[node.kind]);
     this.indent++;
-    if (!this.maybeProcess(node)) {
-      this.writeNode(node);
+    try {
+      if (!this.maybeProcess(node)) {
+        this.writeNode(node);
+      }
+    } catch (e) {
+      e.message += `\n at ${ts.SyntaxKind[node.kind]} in ${this.file.fileName}:`;
```

Comment:
I've added an assignment into message in case it doesn't exist. There's a chance that somebody would just `throw 'foo';`, which is probably also why they are `any` in TS.